app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,21 @@
+class FollowRequestPolicy < ApplicationPolicy
+  attr_reader :user, :follow_request
+
+  def initialize(user, follow_request)
+    @user = user
+    @follow_request = follow_request
+  end
+
+  def create?
+    user
+  end
+
+  def update?
+    user == follow_request.sender || follow_request.recipient
```

Comment:
stay DRY here

Reply:
Might have to repeat `user == follow_request.recipient`

Reply:
Any update on this?

Reply:
corrected to the correct way of writing it